### PR TITLE
Add number of search results

### PR DESCRIPTION
**Pull request by TadejPolajnar**
Closes https://github.com/sledilnik/zdravniki/issues/195

PR adds a number of total search results. 
- It is visible above the doctor list both on mobile and desktop view
- Results on the map are visible only on mobile view since the list and map are not displayed at the same time. I reproduced the zoom toggle design. (Suggestions welcome) 
- If there is 0 results text is hidden since the info alert component is displayed 
- Pluralisation is included in translations 

image
image
image

### CODE REVIEW

File: src/locales/sl.json [hunk 1]
```diff
@@ -27,6 +27,11 @@
     "filter": "Filter",
     "map": "Zemljevid",
     "list": "Seznam",
+    "totalResults_one": "{{count}} rezultat",
+    "totalResults_two": "{{count}} rezultata",
+    "totalResults_three": "{{count}} rezultati",
+    "totalResults_four": "{{count}} rezultati",
+    "totalResults_other": "{{count}} rezultatov",
```
Comment (stefanb) [inline src/locales/sl.json:34]: Je "rezultatov" pravi izraz? IMO bi bilo bolje "zadetkov", "najdenih" ali "ustreznih"....

Rezultat je nekaj dokončnega, npr. ko nekdo najde zdravnika ali obupa nad tem 🙂 ali končni izid tekme, izžrebane številke lota...

Če že hočemo ta izraz, rabi pojasnilo "rezultatov iskanja" oz. "search results"

Podobno v angleščini.

**Comment by stefanb** [comment]
Pluralisation formula is not working properly for 3 and 4, it chooses english translation:
image
Works ok for 1 (eg search for krava), 2 (eg search for kravo) and other cases

File: src/locales/sl.json (hunk 1 above)
Comment (TadejPolajnar) [reply src/locales/sl.json:34]: Drži, hvala! Najbolj primeren izraz se mi zdi `zadetkov`/`hits`. `Rezultatov iskanja` je predolgo, predvsem na koledarju.


Bom popravil asap :)

**Review by jalezi — COMMENTED** [review]
It would be nice if hits are always visible. 

initial
Screenshot 2022-01-06 09 53 07

user scrolls
Screenshot 2022-01-06 09 53 21

**Comment by TadejPolajnar** [comment]
> Pluralisation formula is not working properly or translation key name should be different for 3 and 4, it chooses english translation:
Works ok for 1 (eg search for krava), 2 (eg search for kravo) and other cases

Pluralisation should be working now

SI
image
image
image
image
image

EN
image
image
image

**Comment by TadejPolajnar** [comment]
> It would be nice if hits are always visible.

I'll try to add it but it might be a little bit tricky without refactoring

**Comment by jalezi** [comment]
Just 🤔 Maybe similar as ScrollToTop or I used to have as MUI Badge. I will try to find the commit.

**Review by lukarenko — APPROVED** [review]
Approved - we can improve count always shown later

**Pull request merged**